webpack.config.js

```diff
@@ -4,18 +4,25 @@ const path = require("path");
 const process = require("process");
 const webpack = require("webpack");
 const lodash = require("lodash");
-const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const Webpackbar = require("webpackbar");
-const ZipPlugin = require("zip-webpack-plugin");
 const headersCsp = require("./public/_headersCsp.json");
 
 // load ".env"
 dotenv.config();
 
+const isPrerendering = process.env.SCRIVITO_PRERENDER;
```

Comment:
I think we should move all the prerendering logic to `webpack.config.production.js`. The production script already reads out `SCRIVITO_PRERENDER`. This would also remove more switches in `webpack.config.js`, so that `generateEntry` and `generatePlugins` could be inlined.